core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -167,3 +173,64 @@ fun LpOutlinedTextFieldNumber(
         keyboardOptions = KeyboardOptions.Default.copy(keyboardType = KeyboardType.Number)
     )
 }
+
+/**
+ * [DropdownTextField] it's a textfield to show a list of items inside a box
+ *
+ * @param items it refers to the list that will be given to expand the textfield
+ *
+ * @author Jesus Lopez Gonzalez
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun DropdownTextField(
```

Comment:
I think you need a high order function to get the information of the text field that it is selected